package.json

```diff
@@ -9,7 +9,10 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "nuke": "rm -rf package-lock.json node_modules && npm install && lerna exec \"rm -f package-lock.json\" && lerna clean --yes && lerna bootstrap && lerna exec --stream -- \"test -f package-lock.json || npm install --package-lock-only\"",
-    "test": "karma start --coverage",
+    "test": "npm-run-all test:core test:ui test:dataviz",
+    "test:core": "karma start --package core --coverage",
+    "test:ui": "karma start --package ui --coverage",
+    "test:dataviz": "karma start --package dataviz --coverage",
     "test:watch": "karma start --auto-watch=true --single-run=false",
```

Comment:
Without specifying a package we will run the test for `core` when executing `test:watch`

Reply:
yes, I agree it's a bit weird, was not sure how to handle a missing explicit package, suggestions are very welcome

Reply:
No better solution came to my mind. The only alternative, but not sure if it is better or helps at all  🤷‍♂ , would use a similar approach with `npm-run-all` which would lead to even more commands on `package.json´.